src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -2598,11 +2598,17 @@ bool SvgLoader::header()
         //Override size
         if (loaderData.doc->node.doc.w > 0) {
             w = loaderData.doc->node.doc.w;
-            if (vw < FLT_EPSILON) vw = w;
+            if (vw < FLT_EPSILON) {
+                vw = w;
+                loaderData.doc->node.doc.vw = w;
+            }
```

Comment:
I recommend passing the viewBox(vx, vy, vw, vh) information as an argument of svgSceneBuild(node, vx, vy, vw, vh) when calling svgSceneBuild in SvgLoader::run() (2577 line).
The data in loaderData.doc->node.doc should remain the same as the parsed data.